src/Workspaces/Core/Portable/Rename/ConflictEngine/DeclarationConflictHelpers.cs

```diff
@@ -37,20 +37,23 @@ public static IEnumerable<Location> GetMembersWithConflictingSignatures(IMethodS
                     if (!(conflictingSymbol.PartialDefinitionPart != null && conflictingSymbol.PartialDefinitionPart == renamedMethod) &&
                         !(conflictingSymbol.PartialImplementationPart != null && conflictingSymbol.PartialImplementationPart == renamedMethod))
                     {
-                        foreach (var location in conflictingSymbol.Locations)
-                        {
-                            yield return location;
-                        }
+                        builder.AddRange(conflictingSymbol.Locations);
                     }
                 }
             }
+
+            return builder.ToImmutableAndFree();
```

Comment:
I'm guessing you're asking whether it uses a singleton instance for the empty array.  I don't believe so, since it's a struct.